### PR TITLE
Add default group info

### PR DESCRIPTION
**Pull request by maiqueb**
Fixes issue # (delete if not relevant)

Changes proposed in this pull request:
- fix the acl direction in the deny-all group
- explain the default security group concept, and its rules

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): (please @mention yourself to sign)
@maiqueb 

This pull request needs review by: (please @mention the reviewer if relevant)
@mmirecki 
@dominikholler

### CODE REVIEW

File: source/develop/release-management/features/network/networking-api-security-groups.html.md [hunk 1]
```diff
@@ -509,18 +516,52 @@ It will feature two ACLs, with the following data:
...
+Two rules are created for each direction: one for IPv4, another for IPv6.
+
+It features four ACLs, with the following data:
++ ingress rules
+  * rule #1 - IPv4
+    - priority: ALLOW_PRIORITY
+    - action: ALLOW
+    - direction: to-lport
+    - match: inport == @Default && ip4 && ip4.src == $pg_ip4_Default
+  * rule #1 - IPv6
```
Comment (dominikholler) [inline source/develop/release-management/features/network/networking-api-security-groups.html.md]: What is $pg_ip4_Default?

File: source/develop/release-management/features/network/networking-api-security-groups.html.md [hunk 2]
```diff
@@ -509,18 +516,52 @@ It will feature two ACLs, with the following data:
 + ingress rule
   * priority: DROP_PRIORITY
   * action: drop
-  * direction: from-lport
+  * direction: to-lport
   * match: inport == @<port_group> && ip
 + egress rule
```
Comment (dominikholler) [inline source/develop/release-management/features/network/networking-api-security-groups.html.md]: Can you remind me why the `&& ip` is required?

Comment (maiqueb) [reply source/develop/release-management/features/network/networking-api-security-groups.html.md]: we want to mimic openstack's behavior, which means allowing L2 traffic. 

so we block everything on layers >= 3

File: source/develop/release-management/features/network/networking-api-security-groups.html.md (hunk 1 above)
Comment (maiqueb) [reply source/develop/release-management/features/network/networking-api-security-groups.html.md]: that's the name of the address set where the IPs of the group member's IPs are stored.

I should probably mention that we rely on the remote_group_id parameter to implement this behavior.

File: source/develop/release-management/features/network/networking-api-security-groups.html.md (hunk 2 above)
Comment (dominikholler) [reply source/develop/release-management/features/network/networking-api-security-groups.html.md]: thanks

File: source/develop/release-management/features/network/networking-api-security-groups.html.md (hunk 1 above)
Comment (dominikholler) [reply source/develop/release-management/features/network/networking-api-security-groups.html.md]: yes, I think this would be good

Comment (maiqueb) [reply source/develop/release-management/features/network/networking-api-security-groups.html.md]: @dominikholler Done

File: source/develop/release-management/features/network/networking-api-security-groups.html.md [hunk 3]
```diff
@@ -509,18 +516,55 @@ It will feature two ACLs, with the following data:
...
+  * rule #1 - IPv4
+    - priority: ALLOW_PRIORITY
+    - action: ALLOW
+    - direction: to-lport
+    - match: inport == @Default && ip4 && ip4.src == $pg_ip4_Default
+  * rule #1 - IPv6
+    - priority: ALLOW_PRIORITY
+    - action: ALLOW
+    - direction: to-lport
+    - match: inport == @Default && ip6 && ip4.src == $pg_ip6_Default
```
Comment (mmirecki) [inline source/develop/release-management/features/network/networking-api-security-groups.html.md]: ip4.src?

not  ip6.src?

Comment (maiqueb) [reply source/develop/release-management/features/network/networking-api-security-groups.html.md]: Right you are.
@mmirecki Done

**Pull request merged**